setup.py

```diff
@@ -17,10 +17,11 @@
     )
 
 import platform
+from packaging import version
```

Comment:
I couldn't find a source for this. Is it from [this package](https://pypi.org/project/packaging/)? If so, anyone wanting to install browser-history from source would need to have `packaging` installed?

Reply:
@Samyak2  https://stackoverflow.com/questions/11887762/how-do-i-compare-version-numbers-in-python
Note: packaging has recently been [vendored into setuptools](https://github.com/pypa/setuptools/tree/a94ccbf404a79d56f9b171024dee361de9a948da/pkg_resources/_vendor/packaging).

So I guess everyone should have this be default?

Reply:
@Samyak2 Yes, I think we can explore using `pyproject.toml`